src/objectStorageService/s3DataService.test.ts

```diff
@@ -49,6 +51,131 @@ describe('SUCCESS CASES: Testing create, read, update, delete of resources; vers
         return { success: true, message: 'Resource found', resource: resourceCopy };
     });
 
+    const s3DataService = new S3DataService(DynamoDbDataService, '4.0.1', { enableMultiTenancy: true });
+
+    test('create', async () => {
+        // BUILD
+        // OPERATE
+        const response = await s3DataService.createResource({
+            resourceType: 'Binary',
+            resource: validV4PdfBinary,
+            tenantId: 'tenant1',
+        });
+        // CHECK
+        expect(response).toMatchObject({
+            success: true,
+            message: 'Resource created',
+            resource: {
+                resourceType: 'Binary',
+                contentType: 'application/pdf',
+                presignedPutUrl: 'https://VALID_S3_PUT_URL.com/tenant1/id_1.pdf/VALID_TEMP_CREDENTIAL',
+            },
+        });
+
+        expect(response.resource.data).toBeUndefined();
+        expect(response.resource.content).toBeUndefined();
+        expect(response.resource.id).toBeDefined();
+        expect(response.resource.meta).toBeDefined();

```

Comment:
Nit: may be able to shorten your tests by use [strictEqual](https://jestjs.io/docs/expect#tostrictequalvalue)